ansible_collections/f5networks/f5_modules/plugins/doc_fragments/f5_rest_cli.py

```diff
@@ -51,9 +51,18 @@ class ModuleDocFragment(object):
         default: yes
       timeout:
         description:
+          - transport: rest
           - Specifies the timeout in seconds for communicating with the network device
-            for either connecting or sending commands.  If the timeout is
+            for either connecting or sending commands. If the timeout is
             exceeded before the operation is completed, the module will error.
+          -
+          - Transport: cli
+          - Use the ANSIBLE_PERSISTENT_COMMAND_TIMEOUT environment variable or specify a command_timeout in the ansible.cfg file.
+          -
```

Comment:
revert this change in timeout parameter, this kind of information is best suited for notes section see line 93, add this as a note

Reply:
You want me to move lines 59 and 60 ONLY to the notes section that starts on line 93, correct?